### ISSUE TITLE
[Statement Editor] LS type suggestions are not appearing

### ISSUE THREAD

**Issue by madushajg** [wso2/ballerina-plugin-vscode#395]
**Description:**
Type suggestions coming from the language server are not appearing when using the VSCode plugin with Ballerina version `2201.3.2`.

![Screenshot 2023-01-17 at 21 04 11](https://user-images.githubusercontent.com/26219651/212941683-1a1072b7-b5b3-46f8-a1d6-19e4824c8479.png)

Verified with other versions like `2201.2.4`, `2201.3.1` and type suggestions are coming as expected.

**Affected Versions:**
Ballerina `2201.3.2 RC2`
Plugin `v3.3.6.beta.3`

**Comment by kaumini** [comment]
This issue is due to a change in completion kind of several type suggestions. Previously type suggestions existed as completion kind of 11. Currently type suggestions are provided in 2 completion kinds (11 & 25). Any reason for updating the completion kind of type suggestions? @IMS94 
![Screenshot from 2023-01-18 11-21-14](https://user-images.githubusercontent.com/36252572/213095696-a882cccc-2156-4f37-8f23-2f7f56964bd6.png)

**Comment by malinthar** [comment]
> This issue is due to a change in completion kind of several type suggestions. In previous bal versions type suggestions provided with completion kind of 11. Currently in 2201.3.2, type suggestions are provided in 2 completion kinds (11 & 25). Any reason for updating the completion kind of type suggestions? @IMS94 ![Screenshot from 2023-01-18 11-21-14](https://user-images.githubusercontent.com/36252572/213095696-a882cccc-2156-4f37-8f23-2f7f56964bd6.png)

@kaumini 
With https://github.com/ballerina-platform/ballerina-lang/pull/39037/files#diff-35036535c39f6bddb8b83a67608f69540a95981fe35b32f1c2cc64827501d12aR605, we refactored the completion items provided for types. 

With this change, we have improved the sorting of completion of type completion items by associating a TypeSymbol for the built-in types such as int, string, etc.  Completion items that have an associated TypeSymbol assigned `TypeParamter(25)` kind. That's why the kind has changed from 11 to 25. 

`stream, table, object, map` do not have built-in types. So those do not have associated TypeSymbols. So their completion item kind is not changed (11).  

Those two kinds are shown as follows.

![image](https://user-images.githubusercontent.com/35211477/213110707-be409056-8fcd-4bb0-96f0-4c3a32949296.png)

![image](https://user-images.githubusercontent.com/35211477/213110663-e6e8f06e-2a4b-4d9c-91e0-d92e6944ef68.png)

In summary, 

The built-in types like string, int can be used as a type to define a variable. So those are assigned `TypeParameter(25)` CompletionItemKind.  

But `stream`, `table`, etc can't be used as types without additional type parameters like the `rowtype` in the `table` type. So those are provided the Unit(11) completion item kind indicating that those are langlibs.
@IMS94 Should both have the same `CompletionItemKind` Irrespective of whether it can be used as a type to define a variable. 

From the perspective of the statement editor, what is the expected behaviour @kaumini ?

**Comment by IMS94** [comment]
@malinthar why we took this decision was to benefit from the icon in front of the types (`<T>` icon) as well. Earlier the icon we had in front of such types didn't mean much. But with the new change, types are shown with an identifiable icon. IMO, this is better in UX.
